obstacleAvoidance/obstacleAvoidance.ino

```diff
@@ -0,0 +1,35 @@
+#include <Smartcar.h>
+#include <HardwareSerial.h>
+
+const int forwardSpeed   = 50; 
+const int stoppingSpeed = 0; 
+
+ArduinoRuntime arduinoRuntime;
+
+BrushedMotor leftMotor(arduinoRuntime, smartcarlib::pins::v2::leftMotorPins);
+BrushedMotor rightMotor(arduinoRuntime, smartcarlib::pins::v2::rightMotorPins);
+DifferentialControl control(leftMotor, rightMotor);
+
+SimpleCar car(control);
+
+const int triggerPin           = 6; // D6
+const int echoPin              = 7; // D7
+const unsigned int maxDistance = 100;
```

Comment:
You probably should use a higher maxDistance if the distance you want to stop is also `100`. Otherwise it looks fine.